lib/spack/spack/schema/packages.py

```diff
@@ -69,6 +69,8 @@
     "patternProperties": {r"\w+": {}},
 }
 
+REQUIREMENT_URL = "https://spack.readthedocs.io/en/latest/build_settings.html#package-requirements"
```

Comment:
Want to confirm that you are pointing people to the Concretization Settings and not package requirements at https://spack.readthedocs.io/en/latest/packages_yaml.html#package-requirements, correct?

Reply:
Thanks, I was pointing them at package requirements, but the link changed yesterday :wink:

Reply:
Updated to new URL.